tests/reporting/test_market_unit.py

```diff
@@ -451,9 +439,94 @@ def test_market_migrate_through_one_fork(localFixture, initializeMarket, constan
     assert initializeMarket.getLastReportersDisputeBond() == longToHexString(0)
     assert initializeMarket.getTentativeWinningPayoutDistributionHash() == initializeMarket.getDesignatedReportPayoutHash()
 
+def test_market_disavow_tokens(localFixture, initializeMarket, mockUniverse, mockReportingWindow, mockMarket, mockStakeTokenFactory):
+    stakePayNumeration = [0,0,0,0,numTicks]
+    with raises(TransactionFailed, message="market state has to be AWAITING_FORK_MIGRATION"):
+        initializeMarket.disavowTokens()
+
+    mockUniverse.setForkingMarket(mockMarket.address)
+    mockReportingWindow.setIsForkingMarketFinalized(False)
+    with raises(TransactionFailed, message="market needs stake tokens"):
+        initializeMarket.disavowTokens()
+    
+    mockStakeToken = set_mock_stake_token_value(localFixture, initializeMarket, stakePayNumeration, mockStakeTokenFactory, 10)
+    assert initializeMarket.isContainerForStakeToken(mockStakeToken.address) == True
+    assert initializeMarket.disavowTokens() == True
+    assert initializeMarket.isContainerForStakeToken(mockStakeToken.address) == False
+
+def test_market_first_reporter_comp_check(localFixture, initializeMarket, chain, constants, mockUniverse, mockStakeToken, mockReportingWindow, mockMarket, mockStakeTokenFactory, mockReputationToken):
+    stakePayNumeration = [0,0,0,0,numTicks]
+    existingBalance = chain.head_state.get_balance(tester.a1)
+    mockReputationToken.setBalanceOfValueFor(initializeMarket.address, 1000)
+    with raises(TransactionFailed, message="sender must be a stake token"):
+        initializeMarket.firstReporterCompensationCheck(tester.a1)
+    
+    mockStakeToken2 = set_mock_stake_token_value(localFixture, initializeMarket, stakePayNumeration, mockStakeTokenFactory, 10)
```

Comment:
Any reason why this is named with a `2` suffix?  If not, recommend removing the suffix to avoid confusion in future readers (I stopped and looked around for `mockStakeToken1`).

Reply:
no reason specifically for the '2' only to differentiate it from existing mockStakeToken. I'll make it more clear.